plugin/trino-geospatial/src/test/java/io/trino/plugin/geospatial/TestSpatialPartitioningInternalAggregation.java

```diff
@@ -78,7 +78,7 @@ public void test(int partitionCount)
         Rectangle expectedExtent = new Rectangle(-10, -10, Math.nextUp(10.0), Math.nextUp(10.0));
         String expectedValue = getSpatialPartitioning(expectedExtent, geometries, partitionCount);
 
-        AccumulatorFactory accumulatorFactory = function.bind(Ints.asList(0, 1, 2), Optional.empty());
+        AccumulatorFactory accumulatorFactory = function.bind(Ints.asList(0, 1), Optional.empty());
```

Comment:
> Currently AccumulatorCompiler generated classes accept
inputChannels lists as arguments that are longer than the number of input
paramters, so the test worked anyway.

Should we have check state there?